src/utils.h

```diff
@@ -169,7 +169,7 @@ ts_flags_are_set_32(uint32 bitmap, uint32 flags)
 	return (bitmap & flags) == flags;
 }
 
-static inline uint32
+static inline pg_nodiscard uint32
 ts_set_flags_32(uint32 bitmap, uint32 flags)
 {
 	return bitmap | flags;
```

Comment:
Maybe we should just replace it with `|=`?

Reply:
I also thought about modifying the value directly. But then decided against it to have a function signature that is similar to other PostgreSQL functions like `lappend()`.

Reply:
No, I mean, remove the function altogether. Anyway, with your change it's better.